local-dev-first-start-only.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+function blue() {
+    local TEXT="$1"
+    echo -n "\x1B[1;34m${TEXT}\x1B[0m"
+}
+
+function green() {
+    local TEXT
+    if [ "$1" == "" ]; then
+        TEXT=Done
+    else
+        TEXT="$1"
+    fi
+    echo -n "\x1B[1;32m${TEXT}\x1B[0m"
+}
+
+function red() {
+    local TEXT
+    if [ "$1" == "" ]; then
+        TEXT=Failure
+    else
+        TEXT="$1"
+    fi
+    echo -n "\x1B[31m${TEXT}\x1B[0m"
+}
+
+clear
+
+echo
+echo '########################################################################################################################'
+echo -e " `red 'For less problem while setup: all containers, images and volumes should be deleted'`"
+echo '########################################################################################################################'
+echo
+echo -e " Enter `green '\"y\"'` will remove your docker data"
+echo -e " Enter `green '\"N\"'` (default) will continue without remove"
+echo -e " Enter `green '\"e\"'` or other will exit to the command prompt"
+echo
+echo -ne " Remove all running containers `red '[ (y)es / (N)o / (e)xit ]?'` "; read A
+echo
+
+if [ "$A" == "y" ] || [ "$A" == "Y" ]; then
+	docker rm -f `docker ps -aq`  2>/dev/null
+else
+	if [ "$A" != "n" ] && [ "$A" != "N" ]; then
+		exit 1
+	fi
+fi
+
+echo
+echo -ne " Remove all current volumes `red '[ (y)es / (N)o / (e)xit ]?'` "; read A
+echo
+
+if [ "$A" == "y" ] || [ "$A" == "Y" ]; then
+	yes | docker volume prune     2>/dev/null
+else
+	if [ "$A" != "n" ] && [ "$A" != "N" ]; then
+		exit 1
+	fi
+fi
+
+echo
+echo -ne " Remove all existed images `red '[ (y)es / (N)o / (e)xit ]?'` "; read A
+echo
+
+if [ "$A" == "y" ] || [ "$A" == "Y" ]; then
+	yes | docker system prune -a  2>/dev/null
+else
+	if [ "$A" != "n" ] && [ "$A" != "N" ]; then
+		exit 1
+	fi
+fi
+
+grep -v CMD Dockerfile > Dockerfile-defaultworker
+grep -v CMD Dockerfile > Dockerfile-hydroshare
+
+cat Dockerfile-defaultworker.template >> Dockerfile-defaultworker
+cat Dockerfile-hydroshare.template >> Dockerfile-hydroshare
+
+cp hydroshare/local_settings.template hydroshare/local_settings.py 2>/dev/null
+mkdir -p hydroshare/static/media 2>/dev/null
+mkdir log 2>/dev/null
+chmod -R 777 log 2>/dev/null
+
+echo
+echo '########################################################################################################################'
+echo -e " `blue  ' System is cleaned.'` `red 'DO NOT CLOSE this windows'` `blue ', please open a new terminal window then run the below command:'`"
+echo
+echo -e " `green '     docker-compose -f local-dev.yml up --build '`"
+echo
+echo -e " `blue  ' Waitting a bit long :) util you can see the two textboxes'` `green '\"iRODS is installed and running\"'` `blue 'of '` `green 'data.local.org'` `blue 'and'` `green 'users.local.org'`"
+echo
+echo -e " `blue  ' If you can see the two textboxes, please press '` `green 'ENTER on this window'` `blue 'to continue...'`"
+
```

Comment:
This will probably continue to be a point of confusion for users. At the very least also remind them to cd to the hydroshare directory. I learned many users will just blindly follow instructions and not think about where they are in the pwd

Reply:
This message is buried by the SOLR output. A typical user will be confused
![image](https://user-images.githubusercontent.com/45397838/55340436-af908600-5472-11e9-9e65-4464dfb91c3f.png)